inhouse-ci.jenkinsfile

```diff
@@ -33,7 +33,6 @@ pipeline {
 
     environment {
         MAVEN_OPTS = '-XX:+UnlockExperimentalVMOptions ' +
-                     '-XX:+UseCGroupMemoryLimitForHeap ' +
```

Comment:
Why was this option removed?

Reply:
This experimental flag `UseCGroupMemoryLimitForHeap` has been removed in Java 11.